src/bin/checkrad.in

```diff
@@ -1418,6 +1420,55 @@ sub redback_telnet {
     return 0;
 }
 
+sub juniper_mx_snmp {
+        #receives acct_session
+        my $temp = $ARGV[4];
+        #removes the leading 0s
+        my $clean_temp = int $temp;
+        # Look up community string in naspasswd file.
+        my ($login, $pass) = naspasswd($ARGV[1], 1);
+        if ($login eq '') {
+                $pass = $cmmty_string;
+        } elsif ($login ne 'SNMP') {
+                if ($debug) {
+                        print LOG
+                        "   Error: Need SNMP community string for $ARGV[1]\n";
+                }
+                return 2;
+        }
+                $out=snmpget($ARGV[1], $pass, ".1.3.6.1.4.1.2636.3.64.1.1.1.3.1.3.0.$clean_temp");
```

Comment:
extra tabs/spaces.

Reply:
Just fix these too and few more missing spaces